newrelic-agent/src/main/java/com/newrelic/agent/extension/ExtensionRewriter.java

```diff
@@ -27,7 +27,7 @@
 public class ExtensionRewriter {
 
     static final DependencyRemapper REMAPPER = new DependencyRemapper(ImmutableSet.of("org/objectweb/asm/",
-            "com/google/", "org/apache/commons/"));
+            "com/github/benmanes/caffeine/", "com/google/", "org/apache/commons/"));
```

Comment:
I haven't investigated what this code actually does. Does it make sense to add Caffeine, or should it replace Guava, or...?